vortex-array/src/aliases/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod hash_map;
```

Comment:
`aliases` feels a bit weird, but I don't have a better idea and it does make sense to have a module to contain the type aliases.